### PR TITLE
Second section projects and aboutself

### PR DESCRIPTION
**Pull request by Koshao-sys**
In this section i have worked on the following items;
1. Created the Projects section using CSS grid and flexboxes
2. Created the About Self section
3. Cleared al lint errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
             </ul>
         </div>
     </section>
+    <button class="btn-disabled"><img src="./Images//Disabled.svg" alt=""></button>
+    <section>
+        <h1 class="head-recent">My Recent <br> Works</h1>
+        <span class="recent-bar"></span>
```
Comment (Ibizugbe) [inline index.html:54]: Your project looks a little bit different from the given design, kindly fix it🤗

your project
yourAppearance

the design
figmaAppearance

kindly make proper adjustments to the `spacing ` `<font-size>` and `<font-weight>`

File: index.html [hunk 2]
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
...
+                          <ul>
+                              <li><a href="#">Ruby on rails</a></li>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>SEE PROJECT</button>
+                      </div>
+                </div>
+            </div>
```
Comment (Ibizugbe) [inline index.html]: I think your project cards are looking slightly different from the design😉
your project
projects-app
figma
figmaProject

kindly take note of the spacings and the border and make the required changes🤗

File: index.html [hunk 3]
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
...
+                      <div class="works-image"></div>
+                      <div class="works-inner">
+                          <p>Multi-Post Stories<br>Gain+Glory</p>
+                          <ul>
+                              <li><a href="#">Ruby on rails</a></li>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>SEE PROJECT</button>
```
Comment (Ibizugbe) [inline index.html]: I think your buttons for each projects are not having the required state as given by the design, kindly fix this🤗😉
button states

File: index.html [hunk 4]
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
...
+    <section class="about">
+        <div class="about-myself">
+            <div>
+                <h2>About me</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <span><a href="#">Get My Resume</a></span>
+            </div>
```
Comment (Ibizugbe) [inline index.html]: Your about section looks a little bit different from the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=WZMqc3ufIF1I5Dt6-0). 

your app
yourAbout

the design
designAbt

kindly make the necessary changes. consider the background image, font properties, and spacing.🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [x] Linters are passing✅
- [x] the right github flow method has been used.✅

## Required Changes ♻️
- I suggest you complete all incomplete statements in your readme file, it will give your project a more professional look🤗
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Koshao-sys** [comment]
In this pull request I have worked on below items;
1. Worked on the general appearance to be identical
2. Worked on the hover and active features on the buttons.
3. Adding the background in the second section.

File: index.html [hunk 5]
```diff
@@ -40,13 +40,165 @@ <h1 class="headline-job">I'm a software developer</h1>
...
+                    <div class="works-inner">
+                        <p>Multi-Post Stories<br>Gain+Glory</p>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">css</a></li>
+                            <li><a href="#">javScript</a></li>
+                            <li><a href="#">html</a></li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
```
Comment (roshan-bajgain) [inline index.html:68]: - [ ] As requested by the previous reviewer kindly make changes to the spacing and border. Use this border `1px solid #D0D9D4;` to match the Fimga design
- [ ] Kindly implement in all the cards as requested by the previous reviewer.
### Your design
your

### Figma design
figma

File: index.html [hunk 6]
```diff
@@ -40,13 +40,165 @@ <h1 class="headline-job">I'm a software developer</h1>
...
+                      <div class="works-inner">
+                          <p>Multi-Post Stories<br>Gain+Glory</p>
+                          <ul>
+                              <li><a href="#">Ruby on rails</a></li>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>See Project</button>
+                      </div>
```
Comment (roshan-bajgain) [inline index.html:84]: - [ ] As requested by the previous reviewer please implement the hover effect on your buttons. When I pressed the buttons then the effect did not work as in the Figma design
- [ ] Please implement in all buttons
### Design
fimga

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Koshao-sys, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
- [ ] Please remove the second author in your README file as you did this project alone as this is mentioned by the previous reviewer to updated README file.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Koshao-sys** [comment]
I have further made the changes below;
1. Removed the second Author from my README file.
2. I made adjustments to the cards in the works section.
3. Implemented the hover effect on all buttons.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Koshao-sys , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

- [OPTIONAL] In your readme file the `LICENSE` is missing please add it because it is one of the important things to have in your readme file. 🙂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**